src/ripple/app/tx/impl/Transactor.cpp

```diff
@@ -156,6 +98,8 @@ TER Transactor::payFee ()
     mSourceBalance -= saPaid;
     mTxnAccount->setFieldAmount (sfBalance, mSourceBalance);
 
+    // VFALCO Should we call view().destroyCoins() here as well?
```

Comment:
You cannot. Under certain circumstances, a `tec` return will be re-processed and we'd need to have `destroyCoins` in two places.